### ISSUE TITLE
Diff Colors are hard to read

### ISSUE THREAD

**Issue by cideM** [robertmeta/nofrils#20]
The `nofrils-dark` variant defines [these colors](https://webaim.org/resources/contrastchecker/?fcolor=000080&bcolor=262626) for e.g., `DiffText` and `Normal`, which is pretty hard to read. This actually applies to all diff colors in the dark variant, possibly in other variants too.

**Comment by robertmeta** [comment]
Yeah, it is an issue on many monitors, I need to fix it -- just been busy.
